config/v3/account_closure/send.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Account Closure (request)  Required scope(s): [admin]",
```

Comment:
Can we have a dedicated section for required scopes? title isn't a suitable place.

Reply:
That shouldn't have been committed. it was an earlier approach I was testing, i'll fix it

Reply:
fixed now